lib/env.ts

```diff
@@ -49,6 +49,11 @@ const jacksonOptions: JacksonOption = {
     privateKey: process.env.PRIVATE_KEY || '',
   },
   boxyhqLicenseKey: process.env.BOXYHQ_LICENSE_KEY,
+  noAnalytics:
+    process.env.DO_NOT_TRACK === '1' ||
+    process.env.DO_NOT_TRACK === 'true' ||
+    process.env.BOXYHQ_NO_ANALYTICS === '1' ||
+    process.env.BOXYHQ_NO_ANALYTICS === 'true',
 };
```

Comment:
Should we add the new env variable to the `.env.example` file?

Reply:
Not needed I think, we'll document it instead